script/install/whenever.bash

```diff
@@ -5,6 +5,6 @@ lastupdate=`cat script/update.time` # get time of last update
 lastmodified=`find config/schedule.rb config/backup.yml -printf '%T+\n' | sort -r | head -n1`
 if [[ "$lastupdate" < "$lastmodified" ]] ; then
   echo '$ whenever --update-crontab '`basename \`pwd\``
-  `bundle list whenever`/bin/whenever --update-crontab `basename \`pwd\``
+  whenever --update-crontab `basename \`pwd\``
```

Comment:
```suggestion
  bundle exec whenever --update-crontab `basename \`pwd\``
```
This guarantees the correct version of 'whenever' runs, and is required on fresh installations that use rbenv (which only creates the shims on the next login, and only if configured in .bashrc).

Reply:
Sure